protocols/virtual-fund/readme.md

```diff
@@ -0,0 +1,17 @@
+# Virtual fund off-chain protocol
+
+## Single hop case
+
+Take three actors Alice, Bob and Irene. Given a ledger channel `L` between Alice and Irene and a ledger channel `L'` between Bob and Irene, the clients send and wait on messages as shown in the below sequence diagram in order to create and fund a virtual channel `V`:
```

Comment:
I tend to use `L` and `L'` due to laziness and my math background, but I think it's less common for programmers and maybe it's less clear for permanent documentation.

`L_a` and `L_b` might be better names. What do you think?

Reply:
Sure, seems like an easy fix.

Reply:
On second thought -- L_a is kind of ugly when not rendered in LaTeX. I minded to keep it as is until we hear from someone who actively prefers something else?